modules/cudf/src/series.ts

```diff
@@ -566,6 +566,28 @@ export class AbstractSeries<T extends DataType = any> {
     return Series.new(Column.sequence<U>(opts.size, init, step, opts.memoryResource));
   }
 
+  /**
+   * Constructs a Series with an input filename as source.
+   *
+   * @note If delimiter is omitted, the default is ''.
+   *
+   * @param filepath Path of the input file.
+   * @param delimiter Optional delimiter.
+   *
+   * @returns Series with the sequence
+   *
+   * @example
+   * ```typescript
+   * import {Series} from '@rapidsai/cudf';
+   *
+   * Series.read_text('./inputAsciiFile.txt')
+   * ```
+   */
+  static read_text<U extends Numeric = Int32>(opts: {filepath: string, delimiter?: string}):
+    Series<U> {
+    return Series.new(Column.read_text<U>(opts.filepath, opts.delimiter));
```

Comment:
```suggestion
    return Series.new(Column.read_text<U>(opts.filepath, opts.delimiter ?? ''));
```

Reply:
Actually I am going to accept this suggestion instead of making it required.